src/mjolnir/adminbuilder.cc

```diff
@@ -142,6 +142,7 @@ void buffer_polygon(const polygon_t& polygon, multipolygon_t& multipolygon) {
   auto* outer_ring = geos_helper_t::from_striped_container(polygon.outer());
   std::vector<GEOSGeometry*> inner_rings;
   inner_rings.reserve(polygon.inners().size());
+  std::cout << polygon.inners().size() << std::endl;
```

Comment:
this is crap of course! open for ideas what to do here..

Reply:
fwiw, it doesn't help at all to check for its size for some reason, it'll still enter that condition.. kinda reminds me of python generators, which you can call only once. the second time the same instance is called, they're exhausted.

Reply:
could also make it a LOG_INFO and pretend it's useful info :D

Reply:
@ianthetechie can you do me a favor and try to run the „admin_uk_overrides“ (smth like that) with this branch, but without this log line? My M1 is still set up with Rosetta and I lack the patience to switch, my intel osx compiles fine, but CI keeps crapping out. On intel I was on clang 14. This seems like an outrageous bug to me😄if we can merge this, we can finally get rid of conan and have full support for vcpkg which would be great.

Reply:
Sure thing @nilsnolde 😂 

By "set up with Rosetta" I assume you mean the thing in the docs about configuring your terminal and stuff with Rosetta and running builds there, right? If so then yeah, I ignored that :P

On to the test... it works for me in debug configuration with this log line removed. When I run in release mode, I get an error about `mode_change_count` in `thor/astar_bss.cc` being set but never referenced cuz I guess the assert macro is a no-op in release mode :P Removing that variable and references, it compiles and the `gurka_admin_uk_override` test passes.

Reply:
15 it would seem. Here's my `clang --version` output:

```
Apple clang version 15.0.0 (clang-1500.1.0.2.5)
Target: arm64-apple-darwin23.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

Reply:
And yeah, we're running the same Xcode build (for another day or 2 at least; IIRC Xcode 15.2 just dropped and I'll upgrade soon but currently running 15.1).

Reply:
I'm also running Boost 1.83 FWIW. Here's my CMake output.

<img width="1656" alt="image" src="https://github.com/valhalla/valhalla/assets/2140440/2a142afe-211e-4884-84a9-ec3ac1cbf570">

Reply:
So your env is as close as it can be to CI, yet it passes for you :exploding_head: I'd like to test out GHA, but we only get crap runners there.

Reply:
Hmm, yeah that's really strange. Mine succeeds on that particular test.

I can't actually complete the full test suite though. It craps out in the Python section, probably because I haven't configured my environment perfectly for it. (Worth noting btw that newer versions of python/pip will refuse to sudo install packages globally, so we'll probably need to change how that part of the process to use virtual environments etc.)

The only (highly unsatisfying) explanation that this suggests is something peculiar to running tests in parallel. This doesn't make a lot of sense to me, but I've experienced similar issues in the past (ex: https://github.com/valhalla/valhalla/issues/4048; I think this one may have resolved itself now).

Reply:
> It craps out in the Python section

Yeah I can see in your CMake output that it can only find the Interpreter, for the bindings you'll need a dev installation (or point CMake to the right libs/dirs).

> Worth noting btw that newer versions of python/pip will refuse to sudo install packages globally

We already took care of that:)

I'm very convinced this is a compiler bug, somehow only manifesting in circleci OSX machines :D In the admins we don't do any parallel stuff, we also don't depend on tiles/graphreader or so around that code. But :shrug: , I'm done with this crap for now really.